sdk/servicebus/azure-servicebus/azure/servicebus/_common/utils.py

```diff
@@ -159,3 +178,102 @@ def transform_messages_to_sendable_if_needed(messages):
             return messages._to_outgoing_message()
         except AttributeError:
             return messages
+
+
+@contextmanager
+def send_trace_context_manager(span_name=SPAN_NAME_SEND):
+    span_impl_type = (
+        settings.tracing_implementation()
+    )  # type: Type[AbstractSpan]
+
+    if span_impl_type is not None:
+        with span_impl_type(name=span_name) as child:
+            child.kind = SpanKind.CLIENT
+            yield child
+    else:
+        yield None
+
+@contextmanager
+def _receive_trace_context_manager(receiver, message=None, span_name=SPAN_NAME_RECEIVE):
```

Comment:
nit: extra "_" in the front
```suggestion
def receive_trace_context_manager(receiver, message=None, span_name=SPAN_NAME_RECEIVE):
```

Reply:
It's legacy from when it was on the receiver; I split it off to unify with the other trace logic but didn't rectify the name, so this is a really reasonable call.  Doing it client side since it has to be changed more than just here, but consider this change accepted++